library/core/src/alloc/layout.rs

```diff
@@ -50,7 +50,7 @@ impl Layout {
     ///    must not overflow (i.e., the rounded value must be less than
     ///    or equal to `usize::MAX`).
     #[stable(feature = "alloc_layout", since = "1.28.0")]
-    #[rustc_const_unstable(feature = "const_alloc_layout", issue = "67521")]
+    #[rustc_const_stable(feature = "const_alloc_layout", since = "1.49.0")]
```

Comment:
```suggestion
    #[rustc_const_stable(feature = "const_alloc_layout", since = "1.50.0")]
```
Should this and the others be `1.50.0` now?

Reply:
Yes, yes it should. Good catch.

Reply:
Ok, done.